.github/workflows/test.yml

```diff
@@ -7,6 +7,7 @@ on:
     paths-ignore:
       - 'docs/**'
   pull_request:
+    types: [opened, synchronize, reopened, ready_for_review, converted_to_draft]
```

Comment:
The last element: `converted_to_draft`. Does that mean it will run tests for Draft PRs?

Reply:
It means it will re-evaluate the workflow when you make it a draft. Then it will disable cloud tests, due to the check we have below.